hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -388,4 +396,41 @@ public void manageTssStatus(final State state) {
     public TssKeysAccessor getTssKeysAccessor() {
         return tssKeysAccessor;
     }
+
+    public void processTssEncryptionKeyChecks(
+            @NonNull final UserTxn userTxn,
+            @NonNull final HandleContext handleContext,
+            @NonNull final KeysAndCerts keysAndCerts) {
+        final var readableStoreFactory = new ReadableStoreFactory(userTxn.state());
```

Comment:
We should guard this functionality behind a feature flag. Probably `tss.keyCandidateRoster`.

Reply:
Added feature flag check.